src/main/java/duke/core/Ui.java

```diff
@@ -325,6 +325,28 @@ public boolean confirmTaskToBeDeleted(Task task) {
         }
     }
 
+    /**
+     * It confirms with user on the deletion of a task.
+     * If user confirms, key in 'Y'. Otherwise key in 'N'.
+     *
+     * @param correctedCommand the correctedCommand
+     * @return true if user confirmed the deletion. False otherwise.
+     */
+    public boolean confirmTypoCorrection(String correctedCommand, String userInput) {
+        System.out.println("Ambiguous format! Did you mean(Y/N): \n" + correctedCommand);
+        while (true) {
+            String command = readCommand();
+            if (command.toLowerCase().equals("y")) {
+                return true;
+            } else if (command.toLowerCase().equals("n")) {
+                System.out.println("Proceed with original command: " + userInput);
```

Comment:
Is there any way to move this sequence into the Ui? I feel like its interactivity with the user makes it more Ui-centric and thus should be sectioned in the appropriate class, though I can understand why it is located here as well.

Reply:
I don't quite get what you mean. As this code is implemented in Ui as `confirmTypoCorrection`.
Do you mean by creating a Class to specifically due with this kind of interaction or?